grawlix-cms/_admin/book.view.php

```diff
@@ -546,13 +571,50 @@
 EOL;
 }
 
+// Determine the length of the buffer
+$dateyear = date('Y');
+$datemonth = date('m');
+$dateday = date('d');
+$schedyear = 2020;
+$schedmonth = 1;
+$schedday = 31;
+
+$lastsched = end($book-> pageList);
+$lastsched_id = $lastsched['id'];
+	
+if ( $lastsched_id ) {
+	$lastschedPage = new GrlxComicPage($lastsched_id);
+$schedyear = substr($lastschedPage-> pageInfo['date_publish'],0,4);
+$schedmonth = substr($lastschedPage-> pageInfo['date_publish'],5,2);
+$schedday = substr($lastschedPage-> pageInfo['date_publish'],8,2);
+}
+$datebuffer1 = strtotime(date('Y-m-d', strtotime($dateyear.'-'.$datemonth.'-'.$dateday.' 10:00:00 GMT')));
+$datebuffer2 = strtotime(date('Y-m-d', strtotime($schedyear.'-'.$schedmonth.'-'.$schedday.' 10:00:00 GMT')));
+
+// Find how many days of buffer. 
+// I don't remember what I've coded, but if "+6" is what 
+// I think it is... 
+// TODO:
+// Make the buffer posting time length to be a variable that can be set from the Admin panel.
```

Comment:
We should probably not leave any TODOs in the final code xP

I'm guessing the thing you want to make a variable is the `6`? I don't really understand what's going on here or what the "The buffer should be updated in" text means, some comments or clearer text here would be beneficial. If this is meant to be a hint to the author that they should post some updates, I'm not sure this is necessary at all? I think just showing how many updates they have in their buffer would be sufficient, and it would avoid the need to calculate time at all.